src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -41,48 +41,27 @@ module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
  */
 /* jshint latedef: nofunc */
 function dialogTestHelper(args) {
-    // Sinon doesn't allow wrapping of `window.open` with a spy because `open`
-    // doesn't look like a function in these browsers.
-    var sinonCantWrapWindowOpen = jQuery.browser.msie &&
-        jQuery.browser.versionNumber <= 8;
+    var wymeditor = jQuery.wymeditors(0);
```

Comment:
The way I say it is that this helper is meant to be a general dialog test helper that does a bunch of things which are common throughout the testing of dialogs.

It may, in the future, be extended to do some more things, like, for example, support opening of dialogs by the `dialog` function itself, passing the dialog object. While that will still fit the essence of this helper, if we change it to `tryDialogWithClickAndKeyboardThenAssert`, we would be compelled to change that name again.

The name `dialogTestHelper` currently represents the essence of the helper, well, as I see it. Functional changes within the scope of a dialog test helper will not require renaming.

Now, whether the person who sees this function for the first time is more enlightened by `dialogTestHelper` or `tryDialogWithClickAndKeyboardThenAssert`, that is an interesting question.

It seems that `dialogTestHelper` describes the essence of the function and `try...ThenAssert` describes the form of the function.

Computers don't understand any essence. They are made of form and operate on form. We, on the other hand, are intelligent (even though we may not act like it at times)—we understand essence. Just a thought.

It seems that while naming this function, I have done that out of the concept that I held regarding its essence.

When you come and read along, is `dialogTestHelper` not good enough to help you understand that it helps test dialogs? Would you like the form level of its operation to be present as its name? What about possible future changes, like I mentioned?

Is it not better that the form–level explanations be kept in the docstring, allowing you to understand what this function does if you think you want to know more?

Perhaps we should reach a balance more towards the form, like `openDialogAndAssert`, which is likely to represent the core functionality of this helper, even if changes are made?